docs/.vuepress/config.ts

```diff
@@ -82,6 +82,17 @@ export default defineUserConfig({
 
       `,
     ],
+    [
+      "script",
+      {},
+      `
+      window.addEventListener('load', function() {
+        if(window.location.href == "https://aqwzx-zksync-v2-docs--pr513-fix-contact-links-do-lhwk6zbb.web.app/docs/") {
+          document.querySelector(".homepage").scrollTo(0, document.body.scrollHeight)
+        }
+      });
+      `,
+    ],
```

Comment:
```suggestion
```

Reply:
what was this?

Reply:
To scroll to the bottom, while on the homepage, the script has been updated.

Reply:
So every time a user goes to the homepage, they scroll to the bottom?

Reply:
> So every time a user goes to the homepage, they scroll to the bottom?

Yes, when the homepage loads it automatically scrolls to the bottom, the footer area.

Reply:
so **_every single time_** anyone access our site, the get redirected to the footer?

Reply:
Not redirected per se, the homepage only( https://era.zksync.io/docs/) is automatically scrolled to the bottom of the footer. They can always scroll back up since we have a **scroll-to-top** feature.